### PR TITLE
Fix/add preview in ged menu

### PR DESCRIPTION
**Pull request by LrxGaelle**
https://platform-dev-add-preview-in-ged-menu.bimdata.io

## Description
Added a “preview” item in GED menu

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [ ] My code follows the code style of this project.
- [ ] I have tested my code.
- [x] I want to run the tests for the commits of this PR

### CODE REVIEW

File: src/i18n/lang/fr.json [hunk 1]
```diff
@@ -101,7 +101,8 @@
     "addVersionButtonText": "Ajouter une version",
     "createModelButtonText": "Définir comme modèle",
     "manageAccessButtonText": "Gérer les accès",
-    "openViewerButtonText": "Voir",
+    "openViewerButtonText": "Ouvrir",
```
Comment (Amoki) [inline src/i18n/lang/fr.json:104]: @NicolasRichel does changing French translation work in POEditor?

Comment (NicolasRichel) [reply src/i18n/lang/fr.json:104]: Yes it works.

Comment (NicolasRichel) [reply src/i18n/lang/fr.json:104]: It should also mark translations in other languages as "**fuzzy**" so that we know they need to be updated. ;-)

**Review by Amoki — APPROVED** [review]

**Review by NicolasRichel — APPROVED** [review]

**Review by GabBimdata — APPROVED** [review]

**Pull request merged**